com.unity.uiwidgets/Editor/UIWidgetsEditorPanel.cs

```diff
@@ -91,21 +91,28 @@ public class UIWidgetsEditorPanel : EditorWindow, IUIWidgetsWindow {
         protected virtual void onEnable() {
         }
 
+        protected void SetShowDebugLog(bool showDebugLog) {
```

Comment:
we can simplify the API as DisableDebugLog(), the advantage is two-fold: (1) we give the user a simpler API; (2) the API name tells user that we show debug log by default (which is true in our current design).

in the following line, we should call an API (e.g., _configurations.DisableDebugLog() instead of change the configuration member directly.